pyteal/ast/router_test.py

```diff
@@ -320,12 +355,10 @@ def test_method_config():
         for oc in ON_COMPLETE_CASES
         if str(oc) != str(pt.OnComplete.ClearState)
     ]
+    focg = FullOrderCombinationGen(list(pt.CallConfig), len(ON_COMPLETE_CASES))
```

Comment:
@ahangsu I opened an _optional_ child PR (https://github.com/algorand/pyteal/pull/569) intending to make debugging simpler.  Can you have a look?

Feel welcomed to keep as is.

Reply:
merged